src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -913,7 +907,7 @@ namespace AirLoopHVACDOAS {
             state.dataLoopNodes->Node(this->m_FanInletNodeNum).MassFlowRateMaxAvail = this->SumMassFlowRate;
             state.dataLoopNodes->Node(this->m_FanOutletNodeNum).MassFlowRateMaxAvail = this->SumMassFlowRate;
             state.dataLoopNodes->Node(this->m_FanOutletNodeNum).MassFlowRateMax = this->SumMassFlowRate;
-            if (!this->FanBlowTroughFlag) {
+            if (!this->FanBeforeCoolingCoilFlag) {
                 state.dataLoopNodes->Node(state.dataAirLoop->OutsideAirSys(this->m_OASystemNum).InletNodeNum(1)).MassFlowRateMaxAvail =
                     this->SumMassFlowRate;
             }
```

Comment:
Why does fan placement affect when the OA system MassFlowRateMaxAvail is set at the entrance of the OA system?

Reply:
@rraustad This section is not necessary. I testes 3 cases with different fan placements: Draw, Blow and Middle. No differences were found. This section is removed.